universal-application-tool-0.0.1/test/views/components/TextFormatterTest.java

```diff
@@ -0,0 +1,62 @@
+package views.components;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import j2html.tags.DomContent;
+import j2html.tags.Text;
+import org.junit.Test;
+
+public class TextFormatterTest {
```

Comment:
(blocking) can you add a test for:
```
### Stuff you'll need
> Information about you and each member of your household (Name, Date of birth, Gender, Race/Ethnicity)
>
> Proof of income for all adults. Provide a document that can prove your income. here are some examples:
>* Preferred: 1040 Income Tax form (most recent)
>* Gross paycheck stubs (1 month for all household members before tax income)
```

It's the example from the PR description and is a good test that combines both things.

Reply:
(non-blocking) You could even throw a link somewhere in there to test all three supported features.

Reply:
Done.